src/icons/fileIcons.ts

```diff
@@ -962,5 +962,6 @@ export const fileIcons: FileIcons = {
             ]
         },
         { name: 'slug', fileNames: ['.slugignore'] },
+        { name: 'rescript', fileExtensions: ['.res', '.resi'] }
```

Comment:
Could you please remove the dots before the file extensions? They are not necessary at this place.

Reply:
Woops, my bad. Fixed it! :)